packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -416,6 +423,46 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
       revocationRegistryDefinitionMetadata: {},
     }
   }
+
+  public async fetchIndySchemaWithSeqNo(agentContext: AgentContext, seqNo: number, did: string, type: number) {
+    const indyVdrPoolService = agentContext.dependencyManager.resolve(IndyVdrPoolService)
+
+    const pool = await indyVdrPoolService.getPoolForDid(agentContext, did)
+
+    agentContext.config.logger.debug(`Getting transaction with seqNo '${seqNo}' from ledger '${pool.indyNamespace}'`)
+    const request = new GetTransactionRequest({ ledgerType: type, seqNo })
+
+    agentContext.config.logger.trace(`Submitting get transaction request to ledger '${pool.indyNamespace}'`)
+    const response = await pool.submitReadRequest(request)
+
+    if (response.result.data?.txn.type === '101') {
+      const schema = response.result.data?.txn.data as SchemaType
+
+      const schemaId = getLegacySchemaId(did, schema.data.name, schema.data.version)
+
+      return {
+        schema: {
+          schemaId,
+          attr_name: schema.data.attr_names,
+          name: schema.data.name,
+          version: schema.data.version,
+          issuerId: did,
+          seqNo,
+        },
+        indyNamespace: pool.indyNamespace,
+      }
+    }
+
+    return agentContext.config.logger.error(`Could not get schema from ledger for seq no ${seqNo}'`)
+  }
+}
+
+export interface SchemaType {
```

Comment:
Same here, no need to export

```suggestion
interface SchemaType {
```